libraries/AP_OSD/AP_OSD.cpp

```diff
@@ -223,6 +223,14 @@ const AP_Param::GroupInfo AP_OSD::var_info[] = {
     AP_SUBGROUPINFO2(screen[2], "3_", 29, AP_OSD, AP_OSD_Screen),
     AP_SUBGROUPINFO2(screen[3], "4_", 30, AP_OSD, AP_OSD_Screen),
 
+    // @Param: 2_TYPE
```

Comment:
should be named OSD_TYPE2....to avoid grouping in the second screen parameter list

Reply:
ok param name changed